src/main/java/bobcat/model/TaskList.java

```diff
@@ -0,0 +1,75 @@
+package bobcat.model;
+
+import java.util.ArrayList;
+
+import bobcat.exception.InvalidOpsException;
+import bobcat.model.task.Task;
+
+public class TaskList {
+    private final ArrayList<Task> storage;
+    private int index;
+
+    /**
+     * Constructor for a task list
```

Comment:
Instead of starting with a noun, perhaps you would like to phrase it this way: Constructs an empty task list ?